### DISCUSSION TITLE
Slot for icon in QTimelineEntry

### DISCUSSION THREAD

**Discussion by adarshmadrecha** [quasarframework/quasar#13574] [Ideas / Proposals]
Currently there is slot for Body, Title, Subtitle.
It will be super helpful to have slot for icon as well.

**Comment by MilosPaunovic** [comment]
Check if the `icon` prop does the job for your particular case.

**Reply by adarshmadrecha** [reply]
icon prop is good. But having slot gives more option to customise.
